src/main/java/net/minecraftforge/network/MCRegisterPacketHandler.java

```diff
@@ -32,12 +32,13 @@
 
     public static class ChannelList {
         private Set<ResourceLocation> locations = new HashSet<>();
-
+        private Set<ResourceLocation> remoteLocations = new HashSet<>();
```

Comment:
As the value of `remoteLocations` is always assigned to be an unmodifiable set in `updateFrom`, this should also be initialized to an appropriate empty unmodifiable set (`Set.of()`).

Also, the blank space between the method and the fields should be preserved.

```suggestion
        private Set<ResourceLocation> remoteLocations = Set.of();

```

Reply:
Done!